### DISCUSSION TITLE
Adding support for tortoise-tts to clone voices

### DISCUSSION THREAD

**Discussion by shubham8550** [ThioJoe/Auto-Synced-Translated-Dubs#62] [Ideas]
https://github.com/neonbjb/tortoise-tts 
this tts support voice cloning I wonder if it's possible to modify this repo to use this tts ?

if anyone able to do this it will be great 

 --- Thanks in advance to whoever add this

**Comment by ThioJoe** [comment]
Moving to a discussion. But the problem with tortoise TTS is the creator hasn't released the code required to fine tune it. I've tried it and it kind of sounds like me but not really.
